### ISSUE TITLE
Display search links for related works from a recording page

### ISSUE THREAD

**Issue by y-van-z** [jesus2099/konami-command#246]
Recordings are necessarily added/linked when entering a new release. But works are not. That would be real nice to be able to search for related works from recording pages. That would allow to search work-only databases (such as SACEM) from a recording page.

(I first mentioned it at the end of this https://github.com/jesus2099/konami-command/pull/244#issuecomment-226116075 as a new feature I could likely submit.)

**Comment by y-van-z** [comment]
Basic implementation now exists, it just lacks a configuration box to disable it and disambiguated titles.

**Comment by jesus2099** [comment]
Sorry if I do not have too much time now but my plan already is to enable all token types everywhere. 
A collector will grab all appropriate entities of any page (with its own logic) then all search tools can be provided with their required tokens will be shown. 
You see what I mean? :)

**Comment by jesus2099** [comment]
Like I tried to explain [recently, elsewhere](../issues/187#issuecomment-207776183). ;)

**Comment by y-van-z** [comment]
The merged pull request #238 already enables all token types everywhere for the `mb_ALL-LINKS` script. Your planned collector just has to generate additional tokens (ex : `%release-barcode%`, `%work-composer%`, `%edit-number%`, `%moon-phase%`) and they will be used by search links that require them.

This feature request is independent and complementary. It allows to search for works that do not exist in MusicBrainz database yet. How would you grab an entity that does not exist?

**Comment by jesus2099** [comment]
Oh OK, I see what you mean.
Let me read this again with this in mind but, then, I would have just used `%recording-name%` in some search, no?

**Comment by y-van-z** [comment]
You are right, there only are `%work-name%`-based search links for works. But this code does not make this assumption and allows for other tokens like `%work-latin-script-name%` to be handled as well.

**Comment by jesus2099** [comment]
I’m not sure to understand but have not read it again with that in mind yet, though.
When building some searches, I think people know that they are making a search to find works, starting from the recording name.
Why call it `%work-name%` etc.?

Oh… What you want to achieve is that the work searches (those built up with `%work-…%`) are used on recordings, without defining them _in double_ with `%recording-…%` in the searches?

Well… Maybe… I’ll see… :)

**Comment by jesus2099** [comment]
Then the same could be made for `%release-…%` → `%release-group-…%`, for `%release-group…%` → `%release-…%`, for things like that… maybe.

**Comment by jesus2099** [comment]
And for `%work-…%` → `%recording-…%` maybe too…

**Comment by y-van-z** [comment]
Yes, but that would be only shortcuts:
- a release group usually contains at least one release you can search from,
- a release is necessarily linked to a release group you can search from,
- a work is usually linked to at least one recording you can search from.

Works are usually created from a recording. This feature would allow to easily search for external data on possible related work before creating it.
